docs/usage/self-hosted-configuration.md

```diff
@@ -384,6 +384,13 @@ The above configuration approach will mean the values are redacted in logs like
          "customEnvVariables": {"SECRET_TOKEN": "{{ secrets.SECRET_TOKEN }}"},
 ```
 
+## deleteConfigFile
+
+If set to `true` Renovate tries to delete the self-hosted config file after reading it.
+You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE` environment variable.
+
+The process that runs Renovate must have the correct permissions to delete the config file.
```

Comment:
```suggestion
If set to `true` Renovate tries to delete the self-hosted config file after reading it.

The process that runs Renovate must have the correct permissions to delete the config file.

<!-- prettier-ignore -->
!!! tip
    You can tell Renovate where to find your config file with the `RENOVATE_CONFIG_FILE` environment variable.
```

I think we should:

- first explain how the `deleteConfigFile` works
- then give the tip about the `RENOVATE_CONFIG_FILE` environment variable

So I've moved the paragraphs around, and added a new tip admonition.